content/academy/puppeteer_playwright.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Puppeteer/Playwright course
+description: description
+menuWeight: 7
+category: courses
+paths:
+    - puppeteer-playwright
+---
+
+# [](#puppeteer-playwright-course) Puppeteer/Playwright course
```

Comment:
@mnmkng In my opinion that's confusing and not as good for SEO as **Puppeteer/Playwright** course. **Puppeteer** and **Playwright** definitely have more search frequency than **Headless browser**, and also we aren't really teaching them in depth about headless browsers, we're teaching them in depth about two specific libraries which utilize them.

Reply:
@m-murasovs Opinion?

Reply:
Sure. I guess I just don't like the slash in the title 😄

Reply:
How about `Headless browser course with Puppeteer and Playwright`?

Reply:
Try fitting that into the left menu 😄

Reply:
That's huge

Reply:
> That's huge
eyyyyy 🎉 

If this is an SEO issue, why not just put `headless browser course` into the menu and then mention `puppeteer and playwright` in the meta description? Should still count, no? Especially since we can also set the H1 to the "huge" version, which will have all the keywords